diff --git a/cvat/apps/engine/static/engine/js/annotationUI.js b/cvat/apps/engine/static/engine/js/annotationUI.js
index c69969f..9ef5f80 100644
--- a/cvat/apps/engine/static/engine/js/annotationUI.js
+++ b/cvat/apps/engine/static/engine/js/annotationUI.js
@@ -719,4 +719,4 @@ function translateSVGPos(svgCanvas, clientX, clientY) {
 
 function blurAllElements() {
     document.activeElement.blur();
-}
\ No newline at end of file
+}
diff --git a/cvat/apps/engine/views.py b/cvat/apps/engine/views.py
index 97ab610..97aa006 100644
--- a/cvat/apps/engine/views.py
+++ b/cvat/apps/engine/views.py
@@ -464,14 +464,14 @@ def save_annotation_for_job(request, jid):
     return HttpResponse()
 
 def get_label_name(label_id, jid):
-    url = 'http://127.0.0.1:8080/get/job/{}'.format(jid)
+    url = 'http://ai-master-bigdl-0.sh.intel.com:8080/get/job/{}'.format(jid)
     r = requests.get(url=url)
     resp_dict = r.json()
     return resp_dict['labels'][str(label_id)]
 
 
 def get_label_id(label_name, jid):
-    url = 'http://127.0.0.1:8080/get/job/{}'.format(jid)
+    url = 'http://ai-master-bigdl-0.sh.intel.com:8080/get/job/{}'.format(jid)
     r = requests.get(url=url)
     resp_dict = r.json()
     label_id_getter = {}
@@ -480,7 +480,7 @@ def get_label_id(label_name, jid):
     return int(label_id_getter[label_name])
 
 def get_job_id_from_task_id(tid):
-    url = 'http://127.0.0.1:8080/get/task/{}'.format(tid)
+    url = 'http://ai-master-bigdl-0.sh.intel.com:8080/get/task/{}'.format(tid)
     r = requests.get(url=url)
     resp_dict = r.json()
     return resp_dict['jobs']
diff --git a/cvat/apps/index/views.py b/cvat/apps/index/views.py
index 0db8eda..7d3a0eb 100644
--- a/cvat/apps/index/views.py
+++ b/cvat/apps/index/views.py
@@ -26,18 +26,18 @@ def index(request):
                 tmp = {}
                 tmp['name'] = table.decode('utf-8')
                 tmp['type'] = 'Medicine'
-                tmp['link'] = 'http://127.0.0.1:8080/molview'
+                tmp['link'] = 'http://ai-master-bigdl-0.sh.intel.com:8080/molview'
                 tmp['training_link'] = 'http://davinci-dev994.sh.intel.com:8890/notebooks/Drug.ipynb'
                 tmp['inference_link'] = 'http://davinci-dev994.sh.intel.com:8890/notebooks/Drug_Synthesis.ipynb'
             else:
                 tmp = {}
                 tmp['name'] = table.decode('utf-8')
                 tmp['type'] = 'Medical Imaging'
-                tmp['link'] = 'http://127.0.0.1:8080/?id=71'
+                tmp['link'] = 'http://ai-master-bigdl-0.sh.intel.com:8080/?id=1'
                 tmp['training_link'] = 'http://ai-master-bigdl-0.sh.intel.com:5050/notebooks/jupyter_notebooks/training_script.ipynb'
                 tmp['inference_link'] = 'http://ai-master-bigdl-0.sh.intel.com:5050/notebooks/jupyter_notebooks/training_script.ipynb'
             data.append(tmp)
         
         context = {'tables': data}
 
-        return render(request, 'index/index.html', context=context)
\ No newline at end of file
+        return render(request, 'index/index.html', context=context)
diff --git a/cvat/settings/base.py b/cvat/settings/base.py
index 8e6cbfd..93d7d6f 100644
--- a/cvat/settings/base.py
+++ b/cvat/settings/base.py
@@ -22,7 +22,7 @@ from pathlib import Path
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = str(Path(__file__).parents[2])
 
-ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', 'localhost,127.0.0.1').split(',')
+ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', 'localhost,127.0.0.1,ai-master-bigdl-0.sh.intel.com').split(',')
 INTERNAL_IPS = ['127.0.0.1']
 
 try:
diff --git a/docker-compose.yml b/docker-compose.yml
index 37f747c..fe0deba 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -31,8 +31,8 @@ services:
     ports:
       - "8080:8080"
       - "5050:5050"
-      #- "13345:13345"
-#      - "4040:4040"
+      - "13345:13345"
+      - "4040:4040"
       - "14448:14448"
       - "13338:13338"
     build:
diff --git a/runserver.sh b/runserver.sh
index 7b57a27..eebbe1e 100644
--- a/runserver.sh
+++ b/runserver.sh
@@ -3,7 +3,7 @@ $SPARK_HOME/bin/spark-submit --master "spark://ai-master-bigdl-0.sh.intel.com:70
     --conf 'spark.executor.extraJavaOptions=-Dbigdl.engineType=mkldnn' \
    --conf 'spark.driver.port=13338' \
    --conf 'spark.blockManager.port=14448' \
-   --conf 'spark.driver.host=chengguosu.sh.intel.com' \
+   --conf 'spark.driver.host=ai-master-bigdl-0.sh.intel.com' \
    --conf 'spark.driver.bindAddress=0.0.0.0' \
 	 --driver-memory 4g --class com.intel.aimaster.PredictorServer \
 	--total-executor-cores 48\
@@ -13,4 +13,4 @@ $SPARK_HOME/bin/spark-submit --master "spark://ai-master-bigdl-0.sh.intel.com:70
 	hbase-site.xml \
 	hdfs://ai-master-bigdl-0.sh.intel.com:8020/model_new_helper_API_10.obj \
 	kfb_512_100_test \
-	http://chengguosu.sh.intel.com:13345/updateStatus
+ http://ai-master-bigdl-0.sh.intel.com:13345/updateStatus
diff --git a/supervisord.conf b/supervisord.conf
index acc9bec..ff94175 100644
--- a/supervisord.conf
+++ b/supervisord.conf
@@ -35,6 +35,6 @@ command=%(ENV_HOME)s/wait-for-it.sh cvat_db:5432 -t 0 -- bash -ic "/usr/bin/pyth
 [program:runjupyter]
 command=%(ENV_HOME)s/wait-for-it.sh cvat_db:5432 -t 0 -- bash -ic "jupyter notebook --ip=0.0.0.0 --port=5050 --NotebookApp.token='' --NotebookApp.password='' "
 
-#[program:runsparkserver]
-#command=%(ENV_HOME)s/wait-for-it.sh cvat_db:5432 -t 0 -- bash -ic "sh ~/runserver.sh"
+[program:runsparkserver]
+command=%(ENV_HOME)s/wait-for-it.sh cvat_db:5432 -t 0 -- bash -ic "sh ~/runserver.sh"
 
